### ISSUE TITLE
Kindly asking a full compatibility with Reika's mod.

### ISSUE THREAD

**Issue by lightning02** [DemoXinMC/FortuneOres#5]
https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/739

**Comment by DemoXinMC** [comment]
What MC version is this?  Also, what version of FortuneOres?

**Comment by lightning02** [comment]
FortuneOres v 2.0.5
MC v 1.7.10

**Comment by DemoXinMC** [comment]
There's no reason at all that my chunks shouldn't be registered in the OreDictionary.  Use F3+H to enable Advanced Tooltips and make sure that "oreCopper" or "oreXXX" for whatever other ores is listed. Every other author complains I oredict things too much, first complaint I've had about someone saying I'm not doing it enough.

**Comment by lightning02** [comment]
How can i change the control key in this case?  because h key is set for "hats"

**Comment by DemoXinMC** [comment]
F3+H is a Minecraft hotkey and there's no way to change it. It's unlikely
that Hats is overriding it.

**Comment by lightning02** [comment]
it happarently seems to override...
Can i still check in the config file instead?

**Comment by DemoXinMC** [comment]
Nothing my mod is doing is the problem. I don't know how to advise you if
you can't turn on Advanced Tooltips, which is a vanilla function.

**Comment by lightning02** [comment]
Can this screens helps as much as of F3+H?

![d35092da928561cfe359217641560a37ece233a0819ebff14b pimgpsh_fullsize_distr](https://cloud.githubusercontent.com/assets/11884147/14527833/b5400308-024c-11e6-9503-07891febc152.png)
![4b30b2223b21fcaaabf5abd32a7b30fd353851426a1990a462 pimgpsh_fullsize_distr](https://cloud.githubusercontent.com/assets/11884147/14527839/b9a842f2-024c-11e6-8658-104874fe3b1a.png)

**Comment by DemoXinMC** [comment]
Go ahead and show Reika that. My chunks are registered

**Comment by lightning02** [comment]
Sure, i will :) 
Thanks!

**Comment by lightning02** [comment]
So, in the end, do you plan to work with reika for a possible compatibility with your mods?

**Comment by DemoXinMC** [comment]
There's nothing that can be done on my end. And even if there was, I'm not
maintaining a 1.7.10 codebase anymore. I went to 1.9 and a backport is
available for 1.8.9, but the new code for FortuneOres works completely
differently from the 1.7 code.

**Comment by lightning02** [comment]
Why not mainteining the 1.7.10 version? Reasonably talking almost every mod is still in 1.7.10, it is the version with largest ammount of mods, there are around 1200 mods compared with only 130 available for 1.9. I'm not saying it's a bad idea to move forward but it should be done alongside with the version mostly used by the modding community ( that actually still is 1.7.10).

**Comment by DemoXinMC** [comment]
Because, as I said, the code base is entirely different. The majority of
the mods in community should be on 1.8 as a minimum. Forge for 1.8 is a
year old.

**Comment by lightning02** [comment]
Yes, in theory it should be this way, but actually, even if Forge wish to push forward all the contents are stuck to 1.7.10 mostly, you can see the actual situation also here https://bot.notenoughmods.com/

**Comment by DemoXinMC** [comment]
I'm still no longer interested in maintaining 1.7.10.  3.0 was rewritten
for 1.9 then backlogged to 1.8. If people would move on, so would mods. If
mods would move on, so would people. Its a vicious circle that I'm not
going to contribute to. 1.8.9 is stable for modding, that's where people
should be.

**Comment by lightning02** [comment]
If you are not gonna suppport 1.7.10 anymore then what about let someone other do it giving mod permission? (I mean if someone may offers to do it for you )
